src/status_im2/contexts/shell/jump_to/components/floating_screens/view.cljs

```diff
@@ -24,9 +25,10 @@
      (animation/animate-floating-screen screen-id screen-param))
    [animation id])
   [reanimated/view
-   {:style (style/screen (get @state/shared-values-atom screen-id))}
-   [gesture/gesture-detector
-    {:gesture (shell.gesture/floating-screen-gesture screen-id)}
+   {:style (style/screen (get @state/shared-values-atom screen-id) screen-id)}
+   [(if platform/ios? gesture/gesture-detector rn/view)
+    {:gesture (shell.gesture/floating-screen-gesture screen-id)
```

Comment:
curious if this is throwing a warning in the dev console for adding `gesture` prop to a react-native view?

Reply:
No :D

Reply:
But will added condition, just in case